addons/website_sale/views/templates.xml

```diff
@@ -610,7 +610,7 @@
                                     </t>
                                     <p t-if="True" class="css_not_available_msg alert alert-warning">This combination does not exist.</p>
                                     <div id="add_to_cart_wrap" class="d-inline">
-                                        <a role="button" id="add_to_cart" class="btn btn-primary btn-lg js_check_product a-submit text-uppercase my-1 mr-1 px-5 font-weight-bold flex-grow-1" href="#"><i class="fa fa-shopping-cart mr-2"/>Add to Cart</a>
+                                        <a role="button" id="add_to_cart" class="btn btn-primary btn-lg js_check_product a-submit my-1 mr-1 px-5 font-weight-bold flex-grow-1" href="#"><i class="fa fa-shopping-cart mr-2"/>ADD TO CART</a>
```

Comment:
probably because if you don't want this in uppercase, you can rewrite it with editor. While the deditor don't have any way to toggle the uppercase class :/